docs/root/intro/arch_overview/upstream/load_reporting_service.rst

```diff
@@ -10,6 +10,6 @@ This will initiate a bi-directional stream with a management server. Upon connec
 server can send a :ref:`LoadStatsResponse <envoy_v3_api_msg_service.load_stats.v3.LoadStatsResponse>`
 to a node it is interested in getting the load reports for. Envoy in this node will start sending
 :ref:`LoadStatsRequest <envoy_v3_api_msg_service.load_stats.v3.LoadStatsRequest>`. This is done periodically
-based on the :ref:`load reporting interval <envoy_v3_api_field_service.load_stats.v3.LoadStatsResponse.load_reporting_interval>`
+based on the :ref:`load reporting interval <envoy_v3_api_field_service.load_stats.v3.LoadStatsResponse.load_reporting_interval>`.
 
-Envoy config with LRS can be found at :repo:`/examples/load-reporting-service/service-envoy-w-lrs.yaml`.
+Envoy config with LRS can be found at :ref:`envoy.yaml </start/sandboxes/_include/load-reporting-service/envoy.yaml>`.
```

Comment:
Done.
FWIW, my personal preference is to have it as a link to the github page rather than requiring me to download the file.
context: https://storage.googleapis.com/envoy-pr/2a94f65/docs/intro/arch_overview/upstream/load_reporting_service.html#load-reporting-service-lrs

Reply:
in which case use a `literalinclude` and include the relevant lines (with optional download)

linking to the repo file is inherently broken - its _the_ issue you are currently fixing

Reply:
re above - fwiw, the links dont necessarily break, _if they are maintained exactly correctly_ as they link to a version of the repo - but if the links are not updated when any changes happen they link to a nonexistent file at that hash

mho, its better all round to just ensure the docs builds carry their own artefacts

the annoying bit with building rst is kinda one of its most useful features - ensuring all the internal linkage is correct

Reply:
Agreed that validating docs links/artifacts correctness make sense.